compiler/bin-js_of_ocaml/check_runtime.ml

```diff
@@ -43,6 +43,7 @@ let print_groups output l =
               output_string output (Printf.sprintf "%s\n" name)))
 
 let f (runtime_files, bytecode, target_env) =
+  Generate.init ();
```

Comment:
`Linker.reset ();` might be even better here. It calls`Generate.init` .

Reply:
Done